backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSupportedDisease.java

```diff
@@ -0,0 +1,26 @@
+package gov.cdc.usds.simplereport.db.model;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.IdClass;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@IdClass(DeviceSupportedDiseaseId.class)
```

Comment:
This is a composite id, needed because join tables dont have a primary key and jpa requires a key for every entity